azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+# .NET Desktop
+# Build and run tests for .NET Desktop or Windows classic desktop solutions.
+# Add steps that publish symbols, save build artifacts, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/apps/windows/dot-net
+
+trigger:
+- master
+
+pool:
+  vmImage: 'windows-latest'
```

Comment:
vmImage: 'vs2017-win2016' #'windows-latest'
because of the windows SDK dependency.